### PR TITLE
adds placeholder message property to table component

### LINKED ISSUE #1360: Add placeholder text for empty tables
As suggested in #1342, player peers and rankings should show some placeholder text such as "This player has no peers" or "This player is not ranked on any heroes" when the responses have no data.

### PR DESCRIPTION
**Pull request by JaydenIvanovic**
fixes #1360 

This PR relates to issue #1360 
I have added an extra property on the table component "placeholderMessage" which will be rendered if the data passed to it is empty.

### CODE REVIEW

**Comment by howardchung** [comment]
Do you have a player ID handy with no ranking/peers so I can preview this?

**Comment by JaydenIvanovic** [comment]
Sure do, I found a use for smurf accounts 345191654

File: src/lang/en-US.json [hunk 1]
```diff
@@ -538,7 +538,8 @@
   "pagination_first": "First",
   "pagination_last": "Last",
   "pagination_of": "of",
-  
+  "peers_none": "This player has no peers",
```
Comment (howardchung) [inline src/lang/en-US.json]: Maybe add a "." to the end

**Comment by howardchung** [comment]
preview link:

https://deploy-preview-1389--opendota-stage.netlify.com/players/345191654/peers

LGTM

**Pull request merged**